packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -86965,6 +86974,706 @@
+                    0
+                ]
+            },
+            {
+                "label": "N",
+                "location": [
+                    7.015243103308636,
+                    -0.16556963017329673,
+                    0
+                ]
+            },
+            {
+                "label": "C",
+                "location": [
+                    4.944903635394974,
+                    -1.7177069852364422,
+                    0
+                ]
+            }
+        ],
+        "bonds": [
+            {
+                "type": 1,
+                "atoms": [0, 1]
+            },
+            {
+                "type": 1,
+                "atoms": [1, 2]
+            },
+            {
+                "type": 1,
+                "atoms": [2, 3]
+            },
+            {
+                "type": 1,
+                "atoms": [0, 4]
+            },
+            {
+                "type": 2,
+                "atoms": [2, 5]
+            },
+            {
+                "type": 1,
+                "atoms": [0, 6]
+            },
+            {
+                "type": 1,
+                "atoms": [1, 7],
+                "stereo": 1
+            },
+            {
+                "type": 1,
+                "atoms": [7, 8]
+            },
+            {
+                "type": 1,
+                "atoms": [8, 9]
+            },
+            {
+                "type": 1,
+                "atoms": [9, 10]
+            },
+            {
+                "type": 1,
+                "atoms": [10, 11]
+            },
+            {
+                "type": 1,
+                "atoms": [11, 12]
+            },
+            {
+                "type": 1,
+                "atoms": [12, 13]
+            },
+            {
+                "type": 2,
+                "atoms": [12, 14]
+            },
+            {
+                "type": 1,
+                "atoms": [13, 18]
+            },
+            {
+                "type": 2,
+                "atoms": [18, 17]
+            },
+            {
+                "type": 1,
+                "atoms": [17, 16]
+            },
+            {
+                "type": 1,
+                "atoms": [16, 15]
+            },
+            {
+                "type": 1,
+                "atoms": [15, 19]
+            },
+            {
+                "type": 1,
+                "atoms": [13, 15],
+                "stereo": 1
+            }
+        ],
+        "class": "AminoAcid",
+        "classHELM": "PEPTIDE",
+        "id": "NMethylProline___NMethylProline",
+        "fullName": "N-Methyl-Proline",
+        "alias": "meP",
+        "attachmentPoints": [
+            {
+                "attachmentAtom": 1,
+                "leavingGroup": {
+                    "atoms": [3]
+                },
+                "type": "left"
+            },
+            {
+                "attachmentAtom": 5,
+                "leavingGroup": {
+                    "atoms": [6]
```

Comment:
Leaving group for R2 is incorrect here
<img width="355" alt="image" src="https://github.com/user-attachments/assets/0e56486e-edc6-4daf-a9d3-acdffc546d48">
Moreover, according to the requirements, meP should only have R2, not R1, as well as attachment atoms and leaving group atoms are different